tests/tcp-split-handshake-01-4whs/test.rules

```diff
@@ -0,0 +1,2 @@
+alert tcp any any -> any any (content:"EVIL"; sid:1;)
+alert tcp-stream any any -> any any (content:"EVIL"; sid:2;)
```

Comment:
Is there an expected difference in behavior for these 2 sigs ?

Reply:
no, added `tcp-pkt` variant to make sure they all match